opentelemetry-sdk/src/opentelemetry/sdk/metrics/export/aggregate.py

```diff
@@ -66,9 +66,12 @@ def take_checkpoint(self):
     def merge(self, other):
         with self._lock:
             self.checkpoint += other.checkpoint
-            self.last_update_timestamp = (
-                other.last_update_timestamp or self.last_update_timestamp
-            )
+            if (other.last_update_timestamp is not None and
+                self.last_update_timestamp is not None):
+                if self.last_update_timestamp < other.last_update_timestamp:
+                    self.last_update_timestamp = other.last_update_timestamp
+            elif self.last_update_timestamp is None:
+                self.last_update_timestamp = other.last_update_timestamp
```

Comment:
Might I suggest this alternative?

```suggestion
            if self.last_update_timestamp is None:
                self.last_update_timestamp = other.last_update_timestamp
            elif other.last_update_timestamp is not None:
                if self.last_update_timestamp < other.last_update_timestamp:
                    self.last_update_timestamp = other.last_update_timestamp
```